src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -217,6 +218,52 @@ public async Task Shared()
             Assert.Equal("28", result4.ToObject());
         }
 
+        [Fact]
+        public void RecalcEngine_Symbol_CultureInfo()
+        {
+            var us_Symbols = new SymbolValues();
+            var us_Culture = new CultureInfo("en-US");
+            us_Symbols.AddService(us_Culture);
+            var us_ParserOptions = new ParserOptions() { Culture = us_Culture };
+
+            var fr_Symbols = new SymbolValues();
+            var fr_Culture = new CultureInfo("fr-FR");
+            fr_Symbols.AddService(fr_Culture);
+            var fr_ParserOptions = new ParserOptions() { Culture = fr_Culture };
+
+            var engine = new RecalcEngine();
+
+            Assert.Equal(1.0, (engine.EvalAsync("1.0", CancellationToken.None, options: us_ParserOptions, runtimeConfig: us_Symbols).Result as NumberValue).Value);
+            Assert.ThrowsAsync<InvalidOperationException>(() => engine.EvalAsync("1.0", CancellationToken.None, options: fr_ParserOptions, runtimeConfig: fr_Symbols));
+            Assert.ThrowsAsync<InvalidOperationException>(() => engine.EvalAsync("2,0", CancellationToken.None, options: us_ParserOptions, runtimeConfig: us_Symbols));
+            Assert.Equal(2.0, (engine.EvalAsync("2,0", CancellationToken.None, options: fr_ParserOptions, runtimeConfig: fr_Symbols).Result as NumberValue).Value);
+        }
+
+        [Fact]
+        public void RecalcEngine_Symbol_CultureInfo2()
```

Comment:
>RecalcEngine_Symbol_CultureInfo2 [](http://example.com/codeflow?start=20&length=32)

Can you add a comment:
// Verify that a single IR tree can be re-executed across multiple cultures.